### PR TITLE
Feat/revision history

### PR DESCRIPTION
**Pull request by rainer-exxcellent**

### CODE REVIEW

File: src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/AdvisoryWrapper.java [hunk 1]
```diff
@@ -383,20 +390,81 @@ public AdvisoryWrapper setDocumentTrackingStatus(DocumentTrackingStatus newState
...
+        } else {
+            return null;
+        }
+    }
+
+    public boolean isPrerelease() {
+
+        Semver version = new Semver(this.getDocumentTrackingVersion());
+        return (version.getMajor() < 1)
+                || ((version.getMajor() == 1) && (version.getMinor() == 0) && (version.getPatch() == 0));
```
Comment (cloeser) [inline src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/AdvisoryWrapper.java:461]: Are we really in pre-release if the version is 1.0.0 ?

Comment (rainer-exxcellent) [reply src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/AdvisoryWrapper.java:461]: Yes, we are in the prerelease workflow.
CreateNewCsaf starts with 1.0.1

Comment (cloeser) [reply src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/AdvisoryWrapper.java:458]: When I read "isPrerelease" I think of "are we not in state published yet", which could also be '2.1.1'. Maybe we should rename the method to something like "isInitialVersion" or "isVersionInitialPublicationOrEarlier"?

Comment (rainer-exxcellent) [reply src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/AdvisoryWrapper.java:458]: a rename it to  beforeInitialPublicRelease like defined in the csaf specification

**Review by cloeser — APPROVED** [review]

**Pull request merged**